src/Main.java

```diff
@@ -9,10 +9,12 @@ public static void main(String[] args) {
         boolean continueTranslating = true;
         List<Integer> numberList = new ArrayList<>();
 
+        //filling the numberlist
```

Comment:
Jammer genoeg mogen comments er niet in staan aldus het review-materiaal van Novi.
Wat een beetje apart is aangezien het dingen doet uitleggen en best handig is. 
Dus voor nu lees ik er maar overheen.

Reply:
Hey Collin, volgens mij gaat het niet per se over comments, maar over oude code die door het in comments te zetten nog in je bestand staat